Week2/Pascal's_triangle_4_fixes/src/Triangle.java

```diff
@@ -61,7 +61,37 @@ private static Long C_update(int n, int k) {
 	}
 	
 	public void fix3_LongToBigInteger(int _n) {
-		List l = new List();
+		StringBuilder triangle = new StringBuilder();
+		
+		for (int n = 0; n <= _n; n++) {
+			for (int k = 0; k <= n; k++) {
+				triangle.append(C_update_BigInt(n, k) + " ");
+			}
+			triangle.append("\n");
+		}
+		System.out.println(triangle);
+	}
+	
+	private static BigInteger C_update_BigInt(int n, int k) {
+		Integer max = Integer.max(k, n-k);
+		Integer min = Integer.min(k, n-k);
+		return getMultiplicationAtoB_BigInt(max + 1, n).divide((getFactorial_BigInt(min)));
+	}
+	
+	private static BigInteger getMultiplicationAtoB_BigInt(Integer a, Integer b) {
+		BigInteger result = BigInteger.ONE;;
```

Comment:
double semicolon, consider revising. Also, why do we need to use constants like BigInteger.ONE?